ztp/gitops-subscriptions/argocd/SNOExpansion.md

```diff
@@ -0,0 +1,483 @@
+cnfdf15	[{"status":"True","type":"Ready"}]	{"node-role.kubernetes.io/master":"","node-role.kubernetes.io/worker":""}
+cnfdf16	[{"status":"True","type":"Ready"}]	{"node-role.kubernetes.io/worker":""}
+
+```
+
+### Applying the DU profile
+This procedure assumes the SNO cluster being expanded is provisioned with DU profile, as described in the [README.md](README.md)
+The DU profile is provisioned using `PolicyGenTemplate` resources partitioned into common, group and site-specific manifests.
+For example, for SNO, the git repository linked to the `policies` ArgoCD application will include:
+- [common-ranGen.yaml](example/policygentemplates/common-ranGen.yaml). This template usually contains a set of operator subscriptions and it's unlikely that it should be modified for a worker addition.
+- [group-du-sno-ranGen.yaml](example/policygentemplates/group-du-sno-ranGen.yaml)
+- [example-sno-site.yaml](example/policygentemplates/example-sno-site.yaml)
+- [ns.yaml](example/policygentemplates/ns.yaml)
+- [kustomization.yaml](example/policygentemplates/kustomization.yaml)
+
+The procedure of configuring the DU profile on the worker node is considered as an upgrade. To initiate the upgrade flow, user must update the existing policies, or create an additional ones, and then create a ClusterGroupUpgrade to reconcile the policies in the group of clusters.
+
+#### __Preparations (optional - for ZTP DU profile version 4.11 or below)__
+If the SNO DU profile was deployed using ZTP plugin version 4.11 or below, PTP and SR-IOV operators might be configured to place the daemons only on nodes labelled as 'master'. This configuration, if exists, would prevent PTP / SR-IOV daemons from operating on the worker node. If PTP / SR-IOV daemon node selectors are incorrectly configured on your system, they must be changed before proceeding with the worker DU profile configuration.
+##### __Ensuring PTP daemon selector compatibility__ 
+
+Check PTP operator `daemonNodeSelector`setting:
+```bash
+$ SNO_NAME=<your SNO cluster name here>
+
+$ cat << EOF | oc apply -f -
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
+metadata:
+  name: check-ptp-node-selector
+  namespace: $SNO_NAME
+spec:
+  scope:
+    resource: ptpoperatorconfigs
+    name: default
+    namespace: openshift-ptp
+EOF
+
+$ oc get managedclusterview check-ptp-node-selector -n $SNO_NAME -ojsonpath='{ .status.result.spec }'
+{"daemonNodeSelector":{}}
+```
+
+If the `daemonNodeSelector` is set to `node-role.kubernetes.io/master: ""`, it must be cleared.
+
+**Important: changing the `daemonNodeSelector` will incur temporary PTP synchronization loss**
+
+More information on PTP operator `daemonNodeSelector` is available [in the PTP operator README file](https://github.com/openshift/ptp-operator#ptpoperatorconfig). 
+
+To clear the PTP operator `daemonNodeSelector`, deploy the following manifest on your hub cluster:
```

Comment:
The changes needed have been already merged in It's[1214](https://github.com/openshift-kni/cnf-features-deploy/pull/1214) and [1213](https://github.com/openshift-kni/cnf-features-deploy/pull/1213). The change was in the go code setting the daemon nede selectors to workers unconditionally. This works for all use cases known so far, because masters have the worker label as well.